Source/WebCore/page/scrolling/ScrollingCoordinator.h

```diff
@@ -60,7 +60,7 @@ class ViewportConstraints;
 using FramesPerSecond = unsigned;
 using PlatformDisplayID = uint32_t;
 
-class ScrollingCoordinator : public ThreadSafeRefCounted<ScrollingCoordinator> {
+class ScrollingCoordinator : public ThreadSafeRefCounted<ScrollingCoordinator>, public CanMakeWeakPtr<ScrollingCoordinator> {
```

Comment:
`ThreadSafeRefCounted` + `CanMakeWeakPtr` should be `ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr`